doc/nrf/dm_code_base.rst

```diff
@@ -65,8 +46,7 @@ For example:
 Repository structure
 ********************
 
-In order to manage the combination of repositories and versions, the |NCS| uses :ref:`west <zephyr:west>`, the same tool that Zephyr uses to manage its repository set.
-You can learn more about the reasons behind the introduction of west in :ref:`this section <zephyr:west-history>` of the Zephyr documentation.
+In order to manage the combination of repositories and versions, the |NCS| uses :ref:`west <zephyr:west>`.
```

Comment:
```suggestion
To manage the combination of repositories and versions, the |NCS| uses :ref:`west <zephyr:west>`.
```

Reply:
Flipped the sentence around:
The |NCS| uses :ref:`west <zephyr:west>` to manage the combination of repositories and versions.